server/api/rounds.py

```diff
@@ -710,16 +722,6 @@ def draw_sample(round_id: str, election_id: str):
     round = Round.query.filter_by(id=round_id, election_id=election_id).one()
     election = round.election
 
-    # For rounds after the first round, automatically select a sample size
-    if round.round_num > 1:
-
-        sample_size_options = sample_sizes_module.sample_size_options(election)
-        for round_contest in round.round_contests:
-            if round_contest.contest_id in sample_size_options:
-                round_contest.sample_size = sample_sizes_module.autoselect_sample_size(
-                    sample_size_options[round_contest.contest_id], election.audit_type
-                )
-
```

Comment:
Given this removal, might there be a reason to check that the sample sizes have indeed been computed before trying to draw the sample?

Reply:
I don't think we need to, since the sample size is now a required parameter in the request to create the round (which in turn runs the task to draw the sample).